testchain/node/src/service.rs

```diff
@@ -104,13 +104,6 @@ pub fn new_partial(
 
 	let select_chain = sc_consensus::LongestChain::new(backend.clone());
 
-	// TODO check if this is needed
```

Comment:
https://github.com/pendulum-chain/spacewalk/blob/bda384bcc6db3da88198c714851a261dcdea1063/clients/runtime/src/rpc.rs#L132-L134 The vault calls `manual_seal()` when doing unit/integration testing, so this can be removed.